tests/page-objects/default/enketo/death-report.page.js

```diff
@@ -0,0 +1,66 @@
+const moment = require('moment');
+const genericForm = require('./generic-form.wdio.page');
+
+const PLACE_OF_DEATH = {healthFacility: 'health_facility', home: 'home', other: 'other'};
+
+const FORM = 'form[data-form-id="death_report"]';
+const SUMMARY_SECTION = `${FORM} section[name="/death_report/group_review"]`;
+
+const deathDate = () => $(`${FORM} section[name="/death_report/death_details"] input.ignore.input-small`);
+const deathPlace = (value) => $(`${FORM} input[name="/death_report/death_details/place_of_death"][value="${value}"]`);
+const deathInformation = () => $(`${FORM} textarea[name="/death_report/death_details/death_information"]`);
+const patientNameSummary = () => $(`${SUMMARY_SECTION} ` +
+  `span[data-itext-id="/death_report/group_review/r_patient_details:label"] ` +
+  `span[data-value=" /death_report/patient_display_name "]`);
+const deathDateSummary = () => $(`${SUMMARY_SECTION} span[data-value=" /death_report/death_details/date_of_death "]`);
+const deathInformationSummary = () => $(`${SUMMARY_SECTION} ` +
+  `span[data-value=" /death_report/death_details/death_information "]`);
+
+const setDeathDate = async (value = moment().subtract(1, 'day').format('YYYY-MM-DD')) => {
+  const date = await deathDate();
+  await date.waitForDisplayed();
+  await date.setValue(value);
+};
+
+const selectDeathPlace = async (value = PLACE_OF_DEATH.healthFacility) => {
+  const place = await deathPlace(value);
+  await place.waitForClickable();
+  await place.click();
+};
+
+const setDeathInformation = async (text = 'Test note') => {
+  const note  = await deathInformation();
+  await note.waitForDisplayed();
+  await note.setValue(text);
+};
+
+const getSummaryDetails = async () => {
+  return {
+    patientNameSummary: await patientNameSummary().getText(),
+    deathDateSummary: await deathDateSummary().getText(),
+    deathInformationSummary: await deathInformationSummary().getText(),
+  };
+};
+
+const submitDeathReport = async ({
+  deathDate: deathDateValue = moment().subtract(1, 'day').format('YYYY-MM-DD'),
+  deathPlace: deathPlaceValue = PLACE_OF_DEATH.healtFacility,
+  deathInformation: deathInformationValue = 'Test note',
```

Comment:
Thanks Lore, let me try to explain a little bit better the function.

Those are the parameters of the function, the values that I am asigning there are the default values, that means that if no value is set when the function is call it will use the default value, in this case it will be `PLACE_OF_DEATH.healtFacility` for example.

The reason why I am sending the parameters as a dictionary and not as a regular variable is to give the possibility to send just one parameter when the function is called, using just the name (`deathPlace` for example). I leaned this in a exercise from `exercism` 😄  -> [here](https://exercism.org/tracks/javascript/exercises/queen-attack/solutions/tlepiz) it is in case you want to take a look.

Reply:
So sorry for the confusion. I get it now! Thanks.